store/tikv/coprocessor.go

```diff
@@ -526,8 +611,9 @@ func (worker *copIteratorWorker) run(ctx context.Context) {
 }
 
 // open starts workers and sender goroutines.
-func (it *copIterator) open(ctx context.Context) {
+func (it *copIterator) open(ctx context.Context) error {
```

Comment:
Could you draw a text graph to illustrate how tasks will be transisted?

Reply:
Sure, will do it today. What would better: just add it here, or add it as a comment into code?

Reply:
As a comment is good to me.